assets/js/blocks/mini-cart/edit.tsx

```diff
@@ -237,8 +238,8 @@ const Edit = ( {
 								className="wc-block-editor-mini-cart__drawer-image"
 								src={
 									isRTL()
-										? '/wp-content/plugins/woocommerce-blocks/images/blocks/mini-cart/cart-drawer-rtl.svg'
-										: '/wp-content/plugins/woocommerce-blocks/images/blocks/mini-cart/cart-drawer.svg'
+										? `${ WC_BLOCKS_IMAGE_URL }/blocks/mini-cart/cart-drawer-rtl.svg`
+										: `${ WC_BLOCKS_IMAGE_URL }/blocks/mini-cart/cart-drawer.svg`
```

Comment:
We should be able to remove the initial `/`, as [it's already included](https://github.com/woocommerce/woocommerce-blocks/blob/trunk/assets/js/settings/blocks/constants.ts#L33) in `WC_BLOCKS_IMAGE_URL`.

```suggestion
										? `${ WC_BLOCKS_IMAGE_URL }blocks/mini-cart/cart-drawer-rtl.svg`
										: `${ WC_BLOCKS_IMAGE_URL }blocks/mini-cart/cart-drawer.svg`
```